### PR TITLE
minor improvements 

### PR DESCRIPTION
**Pull request by sa-kib**
Several improvements here and there unrelated to each other. See commit messages for description.

### CODE REVIEW

File: libmusic/src/fft_wrapper.cpp [hunk 1]
```diff
@@ -46,8 +46,10 @@ FFTWrapper::FFTWrapper(freq_hz_t f_low, freq_hz_t f_high, uint32_t sample_rate,
             FFTWrapper(f_low, f_high, BINS_PER_OCTAVE_DEFAULT, sample_rate,
                        win_size, hop_size) {}
 
-void FFTWrapper::Process(td_t td, uint32_t offset)
+void FFTWrapper::Process(const td_t & td, uint32_t offset)
 {
+    std::vector<fd_t> fft_sg;
```
Comment (vmkononenko) [inline libmusic/src/fft_wrapper.cpp]: since `fft_sg` is used only for `LM_PEED` this should probably go under `#ifdef NDEBUG`

Comment (sa-kib) [reply libmusic/src/fft_wrapper.cpp]: this patch is better be dropped

**Comment by sa-kib** [comment]
two more fixes: c58d498 and 2315ced

**Review by vmkononenko — APPROVED** [review]
Thank you!

**Pull request merged**